topgun/common/common.go

```diff
@@ -175,11 +175,16 @@ type BoshInstance struct {
 func StartDeploy(manifest string, args ...string) *gexec.Session {
 	WaitForDeploymentAndCompileLocks()
 
+	var modifiedSuiteName string
+	if suiteName != "" {
+		modifiedSuiteName = "-" + suiteName
+	}
+
 	return SpawnBosh(
 		append([]string{
 			"deploy", manifest,
 			"--vars-store", filepath.Join(tmp, DeploymentName+"-vars.yml"),
-			"-v", "suite='-" + suiteName + "'",
+			"-v", "suite='" + modifiedSuiteName + "'",
```

Comment:
would bosh complain about having a variable set to `""` ?

Reply:
oh nice, it doesn't 😁  

```bash
bosh int ./concourse.yml  -v "suite=''" > a
bosh int ./concourse.yml  -v "suite='foo'" > b
git diff --no-index ./a ./b
```

```diff
diff --git a/./a b/./b
index c1929d6d4..37c1293bb 100644
--- a/./a
+++ b/./b
@@ -28,7 +28,7 @@ instance_groups:
         host_key: ((tsa_host_key))
         log_level: debug
         token_signing_key: ((token_signing_key))
-    release: concourse
+    release: concoursefoo
   name: web
   networks:
   - name: test
@@ -63,7 +63,7 @@ instance_groups:
       log_level: debug
       worker_gateway:
         worker_key: ((worker_key))
-    release: concourse
+    release: concoursefoo
   name: worker
   networks:
   - name: test
@@ -71,7 +71,7 @@ instance_groups:
   vm_type: test
 name: ((deployment_name))
 releases:
-- name: concourse
+- name: concoursefoo
   version: ((concourse_release_version))
 - name: bpm
   version: ((bpm_release_version))
```

Reply:
Good question! That was an assumption of mine.